components/gitpod-db/src/typeorm/migration/1662639748206-CostCenterPaymentStrategy.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const D_B_COST_CENTER = "d_b_cost_center";
+const COL_CREATION_TIME = "creationTime";
+const COL_BILLING_STRATEGY = "billingStrategy";
+
+export class CostCenterPaymentStrategy1662639748206 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, D_B_COST_CENTER, COL_CREATION_TIME))) {
+            await queryRunner.query(
+                `ALTER TABLE ${D_B_COST_CENTER} ADD COLUMN ${COL_CREATION_TIME} varchar(30) NOT NULL, ALGORITHM=INPLACE, LOCK=NONE `,
```

Comment:
Why do we use a `varchar` to store a date here? Would [datetime](https://dev.mysql.com/doc/refman/8.0/en/timestamp-initialization.html) be a better choice?

Reply:
Because we generally use this as a date type (apart from `_lastModified` columns). 
Happy to re-discuss our general strategy. Maybe something to bring up in a team meeting?